### DISCUSSION TITLE
System logs are not sent to Sensor nodes

### DISCUSSION THREAD

**Discussion by itdeveloper50** [Security-Onion-Solutions/securityonion#8876] [Unsupported Versions] [answered]
I have some branches, where sensor nodes are installed in each of them.
Branches will send their system logs to that sensor nodes, and sensor node will forward them to our central Manager.

I succeded in sending logs with wazuh-agent (ossec-agent) to sensor nodes, and then to Manager.
But I cannot send logs with Winlogbeat and Filebeat to sensor nodes.

I have added port 5044 to sensor node's firewall, but netstat is not showing that port,
and Winlogbeat cannot establish connection with sensor node.

Is there a way to send system logs with Winlogbeat and Filebeat to Sensor nodes?

The reason of not sending logs directly to Manager is this:
- Then we plan to assign roles, each branch will have an access to its own sensor.

**Comment by dougburks** [comment, marked as answer]
Sensor nodes do not run Logstash, so you’ll need to configure Winlogbeat/Filebeat to send to your manager or receiver nodes. For more information, please see https://docs.securityonion.net/en/2.3/beats.html.
